src/Symfony/Component/Notifier/Bridge/Expo/ExpoTransport.php

```diff
@@ -93,6 +93,11 @@ protected function doSend(MessageInterface $message): SentMessage
 
         $success = $response->toArray(false);
 
+        $isError = 'error' === $success['data']['status'];
+        if ($isError) {
+            throw new TransportException(sprintf('Unable to post the Expo message: "%s" (%s)', $success['data']['message'], $success['data']['details']['error']), $response);
```

Comment:
Would you have an example of a resulting message?
Maybe a link to the doc about this structure also?

Reply:
Added a link to the API docs in the transport;

And as for an response for an error case:

```json
{
  "data": {
    "status": "error",
    "message": "\"ExponentPushToken[xxxxxxxxxxxxxxxxxxxxxx]\" is not a registered push notification recipient",
    "details": {
      "error": "DeviceNotRegistered",
      "expoPushToken": "ExponentPushToken[xxxxxxxxxxxxxxxxxxxxxx]"
    }
  }
}
```

This can be reproduced with
```shell
curl -H "Content-Type: application/json" -X POST "https://exp.host/--/api/v2/push/send" -d '{
  "to": "ExponentPushToken[xxxxxxxxxxxxxxxxxxxxxx]",
  "title":"hello",
  "body": "world"
}'
```

Any other possible error like invalid auth tokens etc. would be handled by a non 200 http status. AFAIK this is the only case that is still an error but would come back as HTTP/200